plugins/module_utils/sdkmanager.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Stanislav Shamilov <shamilovstas@protonmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import re
+
+from ansible_collections.community.general.plugins.module_utils import cmd_runner_fmt
+from ansible_collections.community.general.plugins.module_utils.cmd_runner import CmdRunner
```

Comment:
That works, but if you would like to make it shorter you can do:
```suggestion
from ansible_collections.community.general.plugins.module_utils.cmd_runner import CmdRunner, cmd_runner_fmt
```
I separated them before to reduce the length of the file, but `cmd_runner_fmt` is imported directly into `cmd_runner`.

Reply:
done